src/unity/toolkits/object_detection/object_detector.cpp

```diff
@@ -346,7 +365,8 @@ std::shared_ptr<MLModelWrapper> object_detector::export_to_coreml(
                                    GRID_SIZE * SPATIAL_REDUCTION,
                                    class_labels.size(),
                                    GRID_SIZE*GRID_SIZE * anchor_boxes().size(),
-                                   std::move(user_defined_metadata));
+                                   std::move(user_defined_metadata), class_labels,
+                                   options); 
```

Comment:
We may as well use std::move for `class_labels` and `options`, the way we have for `user_defined_metadata`. Since these symbols are not being used anywhere after this in the function, it's safe to transfer ownership from them and leave those symbols in undefined states